sos/report/plugins/networking.py

```diff
@@ -132,6 +132,7 @@ def setup(self):
             "ip maddr show",
             "ip -s -s neigh show",
             "ip neigh show nud noarp",
+            "ifconfig -a",
```

Comment:
`ifconfig` has been deprecated for some time in favor of `ip`, which the `networking` plugin already uses to capture interface address information.

Reply:
Thanks Jake,
Will update your comments and send v2 patchset